cli/compose/convert/service.go

```diff
@@ -36,9 +37,19 @@ func Services(
 		if err != nil {
 			return nil, errors.Wrapf(err, "service %s", service.Name)
 		}
-		configs, err := convertServiceConfigObjs(client, namespace, service.Configs, config.Configs)
-		if err != nil {
-			return nil, errors.Wrapf(err, "service %s", service.Name)
+		var configs []*swarm.ConfigReference
+		if len(service.Configs) > 0 {
```

Comment:
What if we tied docker-compose file-version to API version? (also dirty perhaps); i.e. when attempting to use docker-compose file format 1.3.x, produce an error if the supported API version is older then Y.